src/simtools/applications/calculate_incident_angles.py

```diff
@@ -14,39 +13,26 @@
 .. code-block:: console
 
     simtools-calculate-incident-angles \
-        --zenith 20 \
         --off_axis_angles 0 1 2 3 4 \
         --source_distance 10 \
-        --number_of_rays 10000 \
+        --number_of_photons 1000000 \
         --model_version 6.0.0 \
         --telescope MSTN-04 \
         --site North
 
 Command line arguments
 ----------------------
 
-- zenith (float, optional)
-    Zenith angle in degrees. Default: 20.0.
 - off_axis_angles (float, optional)
     One or more off-axis angles in degrees (space-separated). Default: [0.0].
 - source_distance (float, optional)
     Source distance in kilometers. Default: 10.0.
-- number_of_rays (int, optional)
-    Number of star photons to trace per run. Default: 10000.
-- camera_shift (float, optional)
-    Camera shift along optical axis. Reserved for future use. Default: 0.0.
-- overwrite_rdna (flag, optional)
-    Overwrite mirror_reflection_random_angle with 0 deg when not provided.
+- number_of_photons (int, optional)
+    Number of photons of the light source to trace per run. Default: 10000.
 - mirror_reflection_random_angle (float, optional)
-    Explicit mirror_reflection_random_angle in degrees; overrides overwrite_rdna.
-- algn (float, optional)
+    Explicit mirror_reflection_random_angle in degrees.
+- mirror_alignment_random (float, optional)
```

Comment:
Not sure we discussed this (Github doesn't show my old comments). 

Is the application using by default the mirror random refleciton angle and alignment parameters from the database?

Reply:
Should we always use the one from DB or should we allow to overwrite this here?

Reply:
For now, we always use the DB. We discussed this item in last week's meeting: there is a need for a common functionality to overwrite parameters read from the DB. But we definitely don't want individual tools to do this in their own style.